properties_pane/view_level/viewLevelConfig.json

```diff
@@ -216,6 +216,72 @@ making sure that you maintain a proper JSON format.
 			{
 				"propertyKeyword": "pipeline",
 				"hidden": true
+			},
+			{
+				"propertyName": "View Tags",
+				"propertyKeyword": "unityViewTags",
+				"propertyTooltip": "Contains all the tags that have been applied to the view within the metastore. Available only in Unity Catalog",
+				"propertyType": "group",
+				"groupItemLimit": 20,
+				"structure": [
+					{
+						"propertyName": "Key",
+						"propertyKeyword": "unityTagKey",
+						"propertyTooltip": "The key of the tag must be between 1 and 255 UTF-8 characters inclusive, except for the following characters: '.', ',', '-', '=', '/', ':', ' ' (blank space)",
```

Comment:
please see the [HCK-4604](https://hackolade.atlassian.net/browse/HCK-4604) and update accordingly